src/js/controllers.js

```diff
@@ -405,4 +408,15 @@ App.controllers = {
       currency: 'BRL',
     }).format(value);
   },
+  dumpData() {
+    const data = JSON.stringify(App.state.keys.cart)
+    console.log(data)
+    localStorage.setItem(App.state.keys.cart, data)
```

Comment:
are you setting `data` as a `key`?

Reply:
I already made the corrections and forgot to answer, thank you! Yes I put the 'data' and forgot to delete.